int8[] points
#geometry_msgs/Point[] points